scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1587,7 +1587,7 @@ def drawText(self, text, x, y, fontsize, rgb, align="center"):
             tempLabel = Image.new('RGBA', (txt_w, txt_h), (255, 255, 255, 0))
             textdraw = ImageDraw.Draw(tempLabel)
             textdraw.text((0, 0), text, font=font, fill=rgb)
-            w = tempLabel.rotate(90)
+            w = tempLabel.rotate(90, expand=True)
```

Comment:
It works with 2.9 too. The API is the same, it seems that it's just the default behaviour that changed.